internal/synclet/synclet.go

```diff
@@ -30,10 +96,24 @@ func (s Synclet) UpdateContainer(
 	filesToDelete []string,
 	commands []model.Cmd) error {
 
-	fmt.Println("container:", containerId)
-	fmt.Println("bytes in tar:", len(tarArchive))
-	fmt.Println("files to delete:", filesToDelete)
-	fmt.Println("cmds:", commands)
+	err := s.rmFiles(ctx, containerId, filesToDelete)
+	if err != nil {
+		return err
```

Comment:
maybe worth attaching metadata to these errs (i.e. the func that threw them)? but 🤷‍♀️

Reply:
whoops, thanks